Monika After Story/game/cardgames.rpy

```diff
@@ -0,0 +1,4781 @@
+            m 3eka "Let's start with at least 4 cards?{nw}"
+            $ _history_list.pop()
+            menu:
+                m "Let's start with at least 4 cards?{fast}"
+
+                "Alright.":
+                    $ persistent._mas_game_nou_house_rules["start_cards"] = 4
+                    $ ready = True
+
+                "No.":
+                    m 3eua "Then try again."
+
+        elif starting_cards > 25:
+            m 2hub "Ahaha, [player]! How do you think I'll hold all these card?"
+            m 3eua "Let's leave it at 20 cards?{nw}"
+            $ _history_list.pop()
+            menu:
+                m "Let's leave it at 20 cards?{fast}"
+
+                "Alright.":
+                    $ persistent._mas_game_nou_house_rules["start_cards"] = 20
+                    $ ready = True
+
+                "No.":
+                    m 3eua "Then try again."
+
+        else:
+            $ _round = _("round") if persistent._mas_game_nou_house_rules["win_points"] else _("game")
+            m 3eub "Okay, from now we will start each [_round!t] with [starting_cards] cards!"
+            $ persistent._mas_game_nou_house_rules["start_cards"] = starting_cards
+            $ ready = True
+
+    $ del[ready]
+    $ del[starting_cards]
+
+    return
+
+# Explaining rules unlocked after you give Monika the deck
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_nou_explain_rules",
+            prompt="Can you explain NoU rules to me?",
+            category=["games"],
+            pool=True,
+            unlocked=False,
+            conditional="renpy.seen_label('mas_reaction_gift_carddeck')",
+            action=EV_ACT_UNLOCK,
+            rules={"no unlock": None},
+            aff_range=(mas_aff.HAPPY, None)
+        )
+    )
+
+label monika_nou_explain_rules:
+    m 4eub "Even if the game looks complicated at first, it's rather quite simple."
+    m 1eua "I'm sure if we play some more games, you'll get into it."
+    m "We start the game with [persistent._mas_game_nou_house_rules[start_cards]] cards."
+    m "Your goal is to play all your cards before I play mine."
+    m 3eua "To play a card you need to match it by the color or the text on the card with the top card on the discard pile."
+    m "If you can't play card in your turn, you must draw one from the draw pile."
+    m 1eua "You don't have to play it, though."
+    m "After you played a card or skipped your turn, my turn begins. And so on until someone wins."
+    m 4eub "Besides the Number cards, there're also special cards known as Action and Wild cards."
+    m 4eua "You can distinguish an Action card by its symbol and a Wild card by its black color."
+    m 1eua "Those cards can make your opponent skip their turn or even draw some cards."
+    m 1tsu "And by some I mean 12 cards in a row."
+    m 1eua "The Wild cards don't have color which means they can be placed on any card."
```

Comment:
Missing an "a" between have and color, aka The Wild cards don't have a color which means ...